### PR TITLE
Fix apartment typo in sentences-en-_common.yaml

### PR DESCRIPTION
**Pull request by justicefreed**
"appartment" should be "apartment"

### CODE REVIEW

**Comment by tetele** [comment]
@justicefreed it looks like we were not quick enough with the approval and there are now conflicts. Can you please resolve them?

File: sentences/en/_common.yaml [hunk 1]
```diff
@@ -392,9 +392,9 @@ expansion_rules:
   clean: (vacuum|clean)
 
   # Context awareness expansion rules
-  all: "(all [[of] the]|every [single]|each [and every])"
-  are_all: "[<are>] <all>"
-  home: "(home|house|appartment|flat)"
+  all: "(all[[ of] the]|every[ single]|each[ and every])"
```
Comment (tetele) [inline sentences/en/_common.yaml:395]:
```suggestion
  all: "(all [[of] the]|every [single]|each [and every])"
```

File: sentences/en/_common.yaml [hunk 2]
```diff
@@ -392,9 +392,9 @@ expansion_rules:
...
+  are_all: "[<are> ]<all>"
```
Comment (tetele) [inline sentences/en/_common.yaml:396]:
```suggestion
  are_all: "[<are>] <all>
```

**Review by tetele — APPROVED** [review]

File: sentences/en/_common.yaml [hunk 3]
```diff
@@ -393,8 +393,8 @@ expansion_rules:
 
   # Context awareness expansion rules
   all: "(all [[of] the]|every [single]|each [and every])"
-  are_all: "[<are>] <all>"
-  home: "(home|house|appartment|flat)"
+  are_all: "[<are>] <all>
```
Comment (tetele) [inline sentences/en/_common.yaml:396]:
```suggestion
  are_all: "[<are>] <all>"
```

**Comment by tetele** [comment]
I made a mess of it because i am on my phone 😅

**Comment by justicefreed** [comment]
I similarly had done the change in the webui myself and realized I couldn't easily fix it cleanly without cloning it locally.  I can force push a consolidated change into a single clean commit if that's preferable.

**Comment by justicefreed** [comment]
Looks like the tests need to be updated as well.  Let me take a crack at doing that right now and can consolidate into a single clean commit.

**Comment by tetele** [comment]
Thanks @justicefreed and sorry for the long wait

**Pull request merged**